src/main/java/com/github/stephengold/joltjni/VertexList.java

```diff
@@ -177,11 +178,15 @@ public void set(int vertexIndex, Vec3Arg location) {
         set(vertexIndex, location.getX(), location.getY(), location.getZ());
     }
 
+    // *************************************************************************
+    // ConstVertexList methods
+
```

Comment:
The `capacity()`, `empty()`, and `get()` methods should also be moved to this section.